domains/hi-res.localplayer.dev

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "hi-res",
+    "subdomain": "localplayer.dev",
+
+    "owner": {
+        "email": "j3rryfate@hotmail.com"
+    },
+
+    "record": {},   
```

Comment:
Records are required to be added, as if they are not added the subdomain will not get created.